src/main/java/edu/cornell/kfs/krad/antivirus/service/impl/DummyScanResult.java

```diff
@@ -3,15 +3,17 @@
 import edu.cornell.kfs.krad.antivirus.service.ScanResult;
 
 public class DummyScanResult implements ScanResult {
+    
+    private static final boolean PASS_ANTIVIRUS = true;
 
     @Override
     public String getResult() {
-        return ScanResult.Status.PASSED.toString();
+        return PASS_ANTIVIRUS ? ScanResult.Status.PASSED.toString() : ScanResult.Status.FAILED.toString();
```

Comment:
It looks like you hard-coded `PASS_ANTIVIRUS` to `true`, meaning the part after the `:` will never get called. Did you mean to make `PASS_ANTIVIRUS` configurable? You could do so by converting it into an immutable instance variable while delaying its initialization until the constructor call.

Reply:
I did hardcode the value.  For my testing, I just set the value to false to test my code for handling virus failures, and then reset it back to the normal status of true.  I am not sure  it would be beneficial to add a framework for this.

Reply:
In that case, should the changes to this file be undone, to avoid confusion when other developers review this?

In the future, it might be helpful to have some easily configurable dummy/test scanning to help test how our code handles success/failure scenarios. Perhaps a follow-up user story could be used to implement such an improvement.

Reply:
OK, I reverted the changes out and I will create a follow up user story to update DummyAntiVirusServiceImpl

Reply:
I created https://jira.cornell.edu/browse/KFSPTS-29871 to update DummyAntiVirusServiceImpl